components/tabcordion/examples/50-local-tokens.js

```diff
@@ -0,0 +1,116 @@
+/** @jsx jsx */
+
+import { forwardRef } from 'react';
+import { GEL, jsx } from '@westpac/core';
+import { Tab, Tabcordion } from '@westpac/tabcordion';
+import { data } from './_data';
+
+const Panel = forwardRef(({ isLast, isSelected, ...props }, ref) => {
+	const common = {
+		backgroundColor: 'black',
+		color: 'white',
+		padding: 16,
+	};
+	return <div ref={ref} css={common} {...props} />;
+});
+
+const TabItem = ({ isSelected, isJustified, isLast, ...props }) => {
+	const common = {
+		flex: isJustified ? 1 : 0,
+		fontSize: '1rem',
+		marginRight: '0.125rem',
+		outline: 0,
+		padding: '0.875rem 1.125rem',
+		textAlign: 'left',
+		textDecoration: 'none',
+		transition: 'background .3s ease',
+		width: '100%',
+		border: `1px solid black`,
+		backgroundColor: isSelected ? 'black' : 'white',
+		borderTopLeftRadius: 3,
+		borderTopRightRadius: 3,
+		borderBottom: 0,
+		color: isSelected ? 'white' : 'black',
+		cursor: 'pointer',
+
+		'&:last-child': {
+			marginRight: 0,
+		},
+	};
+
+	return <button css={common} {...props} />;
+};
+
+const AccordionLabel = ({ isLast, isSelected, ...props }) => {
+	const common = {
+		alignItems: 'center',
+		backgroundColor: 'white',
+		border: 0,
+		borderTop: `1px solid black`,
+		borderLeft: `1px solid black`,
+		borderRight: `1px solid black`,
+		cursor: 'pointer',
+		display: 'flex',
+		fontSize: '1rem',
+		justifyContent: 'space-between',
+		outline: 0,
+		padding: '0.75rem 1.125rem',
+		position: 'relative',
+		textAlign: 'left',
+		width: '100%',
+	};
+
+	const lastStyles =
+		isLast && !isSelected
+			? {
+					borderBottom: `1px solid black`,
+					borderBottomLeftRadius: 3,
+					borderBottomRightRadius: 3,
+			  }
+			: {};
+
+	return <button css={{ ...common, ...lastStyles }} {...props} />;
+};
+
+function Example({ brand }) {
+	const overridesWithTokens = { ...brand };
+
+	overridesWithTokens['@westpac/tabcordion'] = {
+		Panel,
+		TabItem,
+		AccordionLabel,
+	};
+
+	return (
+		<GEL brand={overridesWithTokens}>
+			<h3>Responsive</h3>
+			<Tabcordion>
+				{data.map(t => (
```

Comment:
I'm actively removing automated layouting from example files so it's easier to read. Let's repeat things so people can the the actual accordion API